server/src/game/mod.rs

```diff
@@ -270,24 +270,17 @@ impl Game {
     }
 
     pub fn game_is_over(&self) -> bool {
-        //find list of all remaining teams, no duplicates, and remove none
-        let remaining_teams: Vec<EndGameCondition> = 
+        if let Some(_) = EndGameCondition::game_is_over(
             PlayerReference::all_players(self)
-                .filter(|p|p.alive(self) && p.end_game_condition(self) != EndGameCondition::None)
-                .map(|p|p.end_game_condition(self))
-                .collect::<std::collections::HashSet<EndGameCondition>>().into_iter().collect::<Vec<EndGameCondition>>();
-
-        //if there are no teams left and multiple wildCards alive then the game is not over
-        if
-            remaining_teams.is_empty() && 
-            PlayerReference::all_players(self)
-                .filter(|p|p.alive(self) && p.role_state(self).role() == role::Role::Wildcard)
-                .count() > 1 
+                .filter_map(|p|
+                    if p.alive(self){Some(p.role(self))}else{None})
```

Comment:
This whitespace is so evil, atp I'd rather have a separate filter and map if you're gonna write your if statements like this. We're not in the interest of saving bytes here